addons/l10n_au/data/account.account.tag.csv

```diff
@@ -0,0 +1,10 @@
+id,name,applicability
+tag_au_tax_sale_10,Tag - au_tax_sale_10,taxes
```

Comment:
Ah, the tags.  To be honest, we have not explored how tags work yet.  I looked at the GB chart and saw they had just labeled each tax with a tag - so I thought for a first push I would put that as is while I refine my understanding of tags and the new reporting.  I will change them, but expect a refinement to this soon.